driver/csiplugin/connectors/rest_v2.go

```diff
@@ -752,6 +850,53 @@ func (s *SpectrumRestV2) CreateS3CacheFileset(ctx context.Context, filesystemNam
 	return nil
 }
 
+func (s *SpectrumRestV2) CreateNodeMappingAFMWithCos(ctx context.Context, exportMapName string, gatewayNodeName string, bucketInfo map[string]string) error {
+	loggerID := utils.GetLoggerId(ctx)
+	klog.V(4).Infof("[%s] rest_v2 CreateNodeMappingAFMWithCos.exportMapName:[%s],gatewayNodeName:[%s], bucketInfo: %v :", loggerID, exportMapName, gatewayNodeName, bucketInfo)
+
+	filesetreq := CreateNodeMapAFMCosRequest{}
+	filesetreq.MapName = exportMapName
+
+	// Extract the hostname without the port
+	parsedURL, err := url.Parse(bucketInfo[BucketEndpoint])
+	if err != nil {
+		return fmt.Errorf("failed to parse endpoint URL %s, error %v", bucketInfo[BucketEndpoint], err)
+	}
+	hostname := parsedURL.Hostname()
+
+	filesetreq.ExportMap = append(filesetreq.ExportMap, fmt.Sprintf(hostname+"/"+gatewayNodeName))
+
+	filesetreq.NoServerResolution = true
```

Comment:
Keeping this as an open comment, so that we don't forget to add --verify-key option, when it's added in GUI.

Reply:
Added comments "Check after GUI fixes" for now. will be verified after the changes available

Reply:
As GUI issue 7542 is fixed, based on that we need to add a change here.

Reply:
This has been closed.